### DISCUSSION TITLE
Date Picker - Time changes after entering #Solved

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#13589] [Help] [answered]
Hello! I´ve recently run into an issue, ever since the time changed. Currently when a user enters a value in my date picker which is set for time only, if they enter a value like 12:00, after pressing enter or clicking outside the box that 12 turns into 11, it happens in all my date time pickers BUT just the very first time they use it, because after correcting it back to normal the picker stops doing said auto correction. I assume it has to do with the timezone after the hour change, but even after trying to set both the table column and the picker to ignore time zones the issue still prevails. I relay on alot of procedures in Mysql such timediff, sum the total hours,..  that's why deleting said columns wasn't a possibility. Also to add up, my auto column date which is in mysql "CURRENT_TIMESTAMP" shows the correct date and time in mysql, but when shown in budibase tables said column date and time has a 1h increase so exporting from budibase i get 2024-04-29 **10**:15:55 but in mysql i have the correct timestamp as 2024-04-29 **09**:15:55.

Any help will be very much appreciated!!

**Comment by ConorWebb96** [comment]
Hey @insomniacj93,

- What version of Budibase are you running?
- Are you self-hosted or using our cloud services?

**Comment by insomniacj93** [comment]
> Hey @insomniacj93,
> 
>     * What version of Budibase are you running?
> 
>     * Are you self-hosted or using our cloud services?

Hey @ConorWebb96, Sorry! Currently I'm self hosted thru docker and on version 2.13.15

I'm on the process of finding the docker-compose.yaml file to check the timezone but only found the .env so far.

**Reply by ConorWebb96** [reply, marked as answer]
Can you try updating to the latest version of Budibase. We made massive changes to the date picker component we used previously recently. A lot of users were experiencing the same issues you've described. 

Hopefully updating to the latest version of Budibase should resolve your issue.

**Comment by insomniacj93** [comment]
> Can you try updating to the latest version of Budibase. We made massive changes to the date picker component we used previously recently. A lot of users were experiencing the same issues you've described.
> 
> Hopefully updating to the latest version of Budibase should resolve your issue.

I wish i could, but sadly i have atm 24 users, and with the update to the limit of users to 20 on the latest versions I'm stuck at the older versions. And the paid version isn't something i can afford as a single person. :(
